packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -490,10 +393,8 @@ provide('legendPosition', toRef(props, 'legendPosition'))
     align-items: center;
     display: flex;
     justify-content: flex-start;
-    padding-bottom: var(--kui-space-60, $kui-space-60);
-    position: v-bind('chartHeaderPosition');
     right: 0;
-    width: v-bind('chartHeaderWidth');
+    width: 100%;
     z-index: 999;
 
     &:hover :deep(.popover-trigger-wrapper) {
```

Comment:
This style doesn't need to be there anymore.

Reply:
done